app/src/main/java/fi/mobiilikehitysprojektir13/newsapp/screens/news/components/NewsCategoryChips.kt

```diff
@@ -40,7 +56,7 @@ fun CategoryChips(onChange: (categories: Set<String>) -> Unit) {
                 }
                 onChange.invoke(selectedCategories.toSet())
             }, label = {
-                Text(text = category.replaceFirstChar { if (it.isLowerCase()) it.titlecase() else it.toString() } ,
+                Text(text = category.replaceFirstChar { if (it.isLowerCase()) it.titlecase() else it.toString() },
```

Comment:
Jos ne on olelisesti jo kaikki lowercase (koska API) niin turhan on tuo if else. Tunge vaan suoran text = it.titlecase() Vaikka se ois UPPER_SNAKE_CASE se silti muutu Titlecase:n